packages/filebrowser/src/browser.ts

```diff
@@ -72,7 +72,11 @@ export class FileBrowser extends SidePanel {
     this.mainPanel.addClass(FILE_BROWSER_PANEL_CLASS);
     this.mainPanel.title.label = this._trans.__('File Browser');
 
-    this.crumbs = new BreadCrumbs({ model, translator });
+    this.crumbs = new BreadCrumbs({
+      model,
+      translator,
+      driveName: 'mydrive'
```

Comment:
I assume this is still a draft?

Reply:
@krassowski It was not a draft but I just converted it into one.

Reply:
I will close this PR. This solution is not the most suited one to display a drive name in a drive filebrowser.